src/index.js

```diff
@@ -0,0 +1,140 @@
+const state = {
+  temperature: {
+    defaultTemp: 65,
+    currentTemp: 65,
+    tempRanges: {
+      hottest: {
+        upperBound: null,
+        lowerBound: 80,
+        landscape: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂',
+      },
+      hot: {
+        upperBound: 79,
+        lowerBound: 70,
+        landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+      },
+      temperate: {
+        upperBound: 69,
+        lowerBound: 60,
+        landscape: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃',
+      },
+      chilly: {
+        upperBound: 59,
+        lowerBound: 50,
+        landscape: '🌲🌲⛄️🌲🍂🌲🍁🌲🌲⛄️🍂🌲',
+      },
+      cold: {
+        upperBound: 49,
+        lowerBound: null,
+        landscape: '🌲🌲⛄️🌲🍂🌲🍁🌲🌲⛄️🍂🌲',
+      },
+    },
+  },
+  sky: {
+    night: '✨✨✨🌙✨✨',
+    sunny: '☁️ ☁️ ☁️ ☀️ ☁️ ☁️',
+    cloudy: '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️',
+    rainy: '🌧🌈⛈🌧💧⛈🌧🌦🌧💧🌧',
+    snowy: '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨',
+  },
+  city: {
+    defaultCity: 'Seattle',
+    inputCity: '',
+  },
+};
+
+const increaseTemp = () => {
+  state.temperature.currentTemp++;
+  setTempDisplay();
+};
+
+const decreaseTemp = () => {
+  state.temperature.currentTemp--;
+  setTempDisplay();
+};
+
+const resetTemp = () => {
+  state.temperature.currentTemp = state.temperature.defaultTemp;
+  setTempDisplay();
+};
+
+// set the class of the temperature so that it corresponds to the temperature
+const setTempDisplay = () => {
+  const tempDisplay = document.getElementById('temp-display');
+  const landscapeDisplay = document.getElementById('ground');
+  let currentTemp = state.temperature.currentTemp;
+
+  tempDisplay.innerHTML = currentTemp;
+
+  for (let climate in state.temperature.tempRanges) {
+    const { upperBound, lowerBound, landscape } =
+      state.temperature.tempRanges[climate];
+    if (upperBound && lowerBound) {
+      if (currentTemp >= lowerBound && currentTemp <= upperBound) {
+        tempDisplay.className = climate;
+        landscapeDisplay.innerHTML = landscape;
+      }
+    } else if (!lowerBound && currentTemp <= upperBound) {
+      tempDisplay.className = climate;
+      landscapeDisplay.innerHTML = landscape;
+    } else if (!upperBound && currentTemp >= lowerBound) {
+      tempDisplay.className = climate;
+      landscapeDisplay.innerHTML = landscape;
+    }
+  }
+};
+
+const selectSky = (event) => {
+  const weather = event.originalTarget.value;
```

Comment:
`originalTarget` works in Firefox but not in any other browsers (https://developer.mozilla.org/en-US/docs/Web/API/Event/originalTarget#browser_compatibility).  I recommend using `target` instead which is more widely implemented.
```suggestion
  const weather = event.target.value;
```